browser-test/src/applicant/application_version_fast_forward.test.ts

```diff
@@ -739,10 +1268,21 @@ class FastForwardApplicantActor {
     programName: string,
     browser: Browser,
   ): Promise<FastForwardApplicantActor> {
-    const context = await browser.newContext()
+    const context = await browser.newContext({
+      recordVideo: {
+        dir: 'tmp/videos/',
```

Comment:
Non-blocking: should this be a constant?

Reply:
Yep. Const in Typescript isn't the same as const in Java. It just means I can't reassign the variable. 

It's not confusing at all :|